### PR TITLE
Tutorial text update

### PR DESCRIPTION
**Pull request by alexc19113800**
Updated tutorial intro conversation to match with document.
Included a simple parsing script (text -> nodes).

Text highlight and most button interactions.
Still lacking:
- detect clicking on animal
- open encyclopedia and notebook

### CODE REVIEW

File: Space-Zoologist/Assets/DialogueEditor/Assets/Scripts/UI/ConversationManager.cs [hunk 1]
```diff
@@ -710,5 +710,15 @@ public void ToggleBacklog()
             bool active = BacklogGameObject.activeSelf;
             BacklogGameObject.SetActive(!active); // toggle backlog
         }
+        Button pingTarget;
+        public void AskForOneTimePing(Button target) {
```
Comment (chaseoqueso) [inline Space-Zoologist/Assets/DialogueEditor/Assets/Scripts/UI/ConversationManager.cs:714]: During the tutorial, the NPC disappeared while instructing the player to perform actions in game. I'm thinking the problem might be one of these methods, please submit a new pull request if you get it figured out.

File: Space-Zoologist/Assets/Scripts/Dialogue/DialogueParser.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using DialogueEditor;
+
+public class DialogueParser : MonoBehaviour
```
Comment (chaseoqueso) [inline Space-Zoologist/Assets/Scripts/Dialogue/DialogueParser.cs:7]: How is this script being used right now? It seems a bit unclear at the moment.

**Review by chaseoqueso — CHANGES_REQUESTED** [review]

File: Space-Zoologist/Assets/DialogueEditor/Assets/Scripts/UI/ConversationManager.cs (hunk 1 above)
Comment (alexc19113800) [reply Space-Zoologist/Assets/DialogueEditor/Assets/Scripts/UI/ConversationManager.cs:714]: This is actually something intended. This acts as a temporary solution for instructing the player to perform an action. The player proceeds in the dialogue after they performed the action. Perhaps we should look for another way to do this (like highlighting)?

File: Space-Zoologist/Assets/Scripts/Dialogue/DialogueParser.cs (hunk 2 above)
Comment (alexc19113800) [reply Space-Zoologist/Assets/Scripts/Dialogue/DialogueParser.cs:7]: This is a parser that helps parse a document like this
https://drive.google.com/drive/u/1/folders/1K4zo5WapP2cGvuXvnIGQPURrI3ceBhZb
and turn it into dialogue nodes (basically, it's helps with making dialogue).

This script is not meant to be used in gameplay. It's a one-time script that creates a dialogue object that can be moved to the dialogue manager.

**Comment by alexc19113800** [comment]
Made a new PR request to merge into Staging.

**Pull request closed without merging**